freqtrade/templates/sample_strategy.py

```diff
@@ -77,7 +77,7 @@ class SampleStrategy(IStrategy):
     exit_short_rsi = IntParameter(low=1, high=50, default=30, space='buy', optimize=True, load=True)
 
     # Number of candles the strategy requires before producing valid signals
-    startup_candle_count: int = 30
+    startup_candle_count: int = 170
```

Comment:
I assume this is the outcome of running the analysis against this strategy?

Reply:
Kinda. It's supposed to be 200 actually. Gonna update it